packages/liquidator/src/constants.js

```diff
@@ -0,0 +1,9 @@
+// As defined in 1inch
+const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
I think you should make this clear that this is a 1inch defined ETH_ADDRESS and rename this var to `ONE_ETH_ADDRESS`

Reply:
This isn't unique to 1inch, its also used on protocols like Aave, Uniswap, behind the scenes for where they swap `ETH` for `WETH`

Reply:
ah ok, can you rename to something to signify why it is not the 0x0 address as expected?

Reply:
You want me to rename `ETH_ADDRESS` to something else?

Reply:
When I see `ETH_ADDRESS` I think of 0x0, so I think you could come up with a more specific or detailed name for the 0xEeeE address

Reply:
What do you suggest?

Reply:
`ALTERNATIVE_ETH_ADDRESS`